src/fswAlgorithms/transDetermination/ephemDifferenceWithUncertainty/ephemDifferenceWithUncertainty.cpp

```diff
@@ -48,3 +48,35 @@ void EphemDifferenceWithUncertainty::Reset(uint64_t CurrentSimNanos)
 void EphemDifferenceWithUncertainty::UpdateState(uint64_t CurrentSimNanos)
 {
 }
+
+/*! Set the covariance of the base object
+    @param Eigen::MatrixXd covariance
+    @return void
+    */
+void EphemDifferenceWithUncertainty::setCovarianceBase(const Eigen::MatrixXd stateCovariance){
+    this->covarianceBase.resize(stateCovariance.rows(), stateCovariance.cols());
+    this->covarianceBase << stateCovariance;
+}
+
+/*! Get the covariance of the base object
+    @return Eigen::MatrixXd covariance
+    */
+const Eigen::MatrixXd EphemDifferenceWithUncertainty::getCovarianceBase() const {
+    return this->covarianceBase;
+}
+
+/*! Set the covariance of the base object
```

Comment:
:large_blue_circle: What is the base object? Can we say what the object is rather than using the word object. Similar further down.

Reply:
Replaced it with "celestial object" and added example of asteroid for base and example of spacecraft for secondary.